flake.nix

```diff
@@ -69,115 +63,75 @@
             };
           }
         )
-        ({ config, pkgs, ... }: {
-          packages.plutip-core.components.tests."plutip-tests".build-tools = [
-            config.hsPkgs.cardano-cli.components.exes.cardano-cli
-            config.hsPkgs.cardano-node.components.exes.cardano-node
-          ];
-          packages.plutip-core.components.exes.local-cluster = {
-            pkgconfig = [ [ pkgs.makeWrapper ] ];
-            postInstall = with pkgs; ''
-              wrapProgram $out/bin/local-cluster \
-                --prefix PATH : "${lib.makeBinPath [
-                  config.hsPkgs.cardano-cli.components.exes.cardano-cli
-                  config.hsPkgs.cardano-node.components.exes.cardano-node
-                ]}"
-            '';
-          };
-        })
-      ];
-      extraSources = [
```

Comment:
`extraSources` does not exist in the iohk haskell-nix, so everything here has to come either from CHap, cabal.project or a flake input